punjab/stream.py

```diff
@@ -13,7 +13,7 @@ def _onStartElement(self, name, attrs):
             qname = ('', name)
 
         # Process attributes
-        for k, v in attrs.items():
+        for k, v in list(attrs.items()):
```

Comment:
Is `list` necessary? It does not seem to match other changes.

Reply:
Your right. It's not necessary.